src/main/java/org/littleshoot/proxy/impl/ProxyConnection.java

```diff
@@ -421,7 +422,7 @@ boolean shouldExecuteOnEventLoop() {
      */
     protected void aggregateContentForFiltering(ChannelPipeline pipeline,
             int numberOfBytesToBuffer) {
-        pipeline.addLast("inflater", new HttpContentDecompressor());
+        pipeline.addLast("inflater", new BrotliHttpContentDecompressor());
```

Comment:
the name was general before and now it is brotli specific, isn't it now handing 2 different compression algorithms?

Reply:
can it be `GeneralHttpContentDecompressor`?